extensions/neo4j/runtime/src/main/java/io/quarkus/neo4j/runtime/Neo4jConfiguration.java

```diff
@@ -46,4 +53,57 @@
         @ConfigItem(defaultValue = "false")
         public boolean disabled = false;
     }
+
+    @ConfigGroup
+    static class Pool {
+
+        /**
+         * Flag, if metrics are enabled.
+         */
+        public boolean metricsEnabled = false;
+
+        /**
+         * Flag, if leaked sessions logging is enabled.
+         */
+        public boolean logLeakedSessions = false;
+
+        /**
+         * The maximum amount of connections in the connection pool towards a single database.
+         */
+        @ConfigItem(defaultValue = "100")
+        public int maxConnectionPoolSize;
+
+        /**
+         * Pooled connections that have been idle in the pool for longer than this timeout will be tested before they are used
+         * again. The value {@literal 0} means connections will always be tested for validity and negative values mean
+         * connections
+         * will never be tested.
+         */
+        @ConfigItem(defaultValue = "PT-0.001S")
```

Comment:
With a custom duration converter we have in place the prefix `PT` is not needed.

Reply:
I noticed it, but put this in the standard format on purpose.

It's nice to be able to do this, but using the ISO format makes it easier to copy properties from one project to another, where maybe such a converter is not in place.

Reply:
Well, the Quarkus .properties is specific and you always have this converter in place.

As mentioned above, `-1` should work and would be much more readable.

Reply:
See above. 

I removed the prefix now and hope I found a good compromise for the guide, that allows sticking with the same default as the driver.